.github/setup-environment/action.yaml

```
@@ -0,0 +1,22 @@
+      with:
+        compiler: gcc
+
+    - name: Install CMake
+      uses: jwlawson/actions-setup-cmake@v1
+      with:
+        cmake-version: '3.29.0'
+
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y zlib1g-dev zlib1g
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)